index.js

```diff
@@ -0,0 +1,94 @@
+const url = "http://localhost:3000/players"
+const playerContainer = document.querySelector('div.player-container')
+const playerForm = document.querySelector('#new-player-form')
+
+document.addEventListener('DOMContentLoaded', () => {
+
+// Deliverable 1: Fetch Players
+   fetch(url)
+  .then(response => response.json())
+  .then(playersArray => {
+    //   console.log(playersArray)
+      allPlayer(playersArray)
+    })
+
+      function allPlayer(playersArray)  {
+      playersArray.forEach(onePlayer)}
+
```

Comment:
Work on indenting/formatting here, it's important to keep your code clean and easy to read so when there's a bug or when you need to make changes to a function, it makes your life easier!

```js
function allPlayer(playersArray) {
    playersArray.forEach(onePlayer)
}
```